pkg/sorter/sorter.go

```diff
@@ -102,7 +102,7 @@ type Position struct {
 
 // Valid indicates whether the position is valid or not.
 func (p Position) Valid() bool {
-	return p.StartTs != 0 && p.CommitTs != 0
```

Comment:
Why do we remove start ts?

Reply:
In some tests transactions' `StartTs` can be 0, so just check `CommitTs` is OK.